### DISCUSSION TITLE
refactor drainpipe into a lib to make it easier to extend into other apps

### DISCUSSION THREAD

**Discussion by codegod100** [likeandscribe/frontpage#180] [Ideas]
Here is my diff https://github.com/likeandscribe/frontpage/compare/main...codegod100:frontpage:push-yrpqrtswktzz

**Comment by WillCorrigan** [comment]
Hey! We may have been a bit over zealous with requesting discussions before every PR, we don't want to stifle people that just want to contribute! If you'd like to open all these again we'd be happy to look over them. I've put a rudimentary PR template in place for the time being.

**Reply by codegod100** [reply]
opened PR on this one
